.github/workflows/update-version.yml

```diff
@@ -0,0 +1,37 @@
+name: Update Version
+on:
+  workflow_dispatch:
+    inputs:
+      version: 
+        description: 'New version to use for the Places extension. Example: 3.0.0'
+        required: true
+
+      core-dependency:
+        description: '[Optional] Update Core dependency in pom.xml. Example: 3.0.0'
+        required: false
+
+       edge-dependency:
```

Comment:
There is an extra space here before `edge-dependency` which will make the workflow fail

Reply:
Its a soft dependency and not a hard dependency. WIll remove them
Thanks